Formula/rdkit.rb

```diff
@@ -0,0 +1,71 @@
+class Rdkit < Formula
+  desc "Interpreted, interactive, object-oriented programming language"
+  homepage "https://rdkit.org/"
+  url "https://github.com/rdkit/rdkit/archive/Release_2020_09_4.tar.gz"
+  sha256 "9e734ca8f99d8be1ef2ac51efb67c393c62e88b98cfa550d6173ce3eaa87b559"
+  license "BSD-3-Clause"
+  head "https://github.com/rdkit/rdkit.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "eigen"
+  depends_on "freetype"
+  depends_on "numpy"
+  depends_on "postgresql@12"
+  depends_on "py3cairo"
+  depends_on "python@3.9"
+
+  def install
+    ENV.cxx11
+    ENV.libcxx
+    ENV.append "CFLAGS", "-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+    ENV.append "CXXFLAGS", "-Wno-parentheses -Wno-logical-op-parentheses -Wno-format"
+
+    # Get Python location
+    python_executable = Formula["python@3.9"].opt_bin/"python3"
+    py3ver = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    py3prefix = Formula["python@3.9"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+    py3include = "#{py3prefix}/include/python#{py3ver}"
+
+    # set -DMAEPARSER and COORDGEN_FORCE_BUILD=ON to avoid conflicts with some formulae i.e. open-babel
+    args = std_cmake_args + %W[
+      -DRDK_INSTALL_INTREE=OFF
```

Comment:
Try adding `-DCMAKE_INSTALL_RPATH=#{opt_lib}` here.

Reply:
Thank you, @carlocab. The issue was solved!